packages/tutorial/.gitignore

```diff
@@ -0,0 +1,2 @@
+public/index.html
+src
```

Comment:
@Guria Should it not be /src?
Like this all `parts/xx/src/` are excluded from github.